packages/plugin/src/components/ApiItem.tsx

```diff
@@ -74,6 +76,37 @@ export default function ApiItem({ readme: Readme, route }: ApiItemProps) {
 		[nextItem, prevItem],
 	);
 
+	// Add @reference categories.
+	const referenceCategories: Record<string, { title: string; children: number[] }> = {};
```

Comment:
This data should probably be built before even making it to the react side of things.

Reply:
Fixed. Please re-review.